dcm4che-conf/dcm4che-conf-core/src/test/java/org/dcm4che3/conf/core/refindexing/RefIndexingTest.java

```diff
@@ -11,22 +10,22 @@
 import org.junit.Before;
 import org.junit.Test;
 
-import java.util.Arrays;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 
 public class RefIndexingTest {
 
     private HashMap<String, Path> uuidToSimplePathCache;
     private Configuration configuration;
+    private Configuration lowerLevelConfiguration;
     private DefaultBeanVitalizer vitalizer;
 
     @Before
     public void init() {
-        configuration = new InMemoryConfiguration();
-        uuidToSimplePathCache = new HashMap<String, Path>();
-        configuration = new ReferenceIndexingDecorator(configuration, uuidToSimplePathCache);
+        lowerLevelConfiguration = new InMemoryConfiguration();
```

Comment:
Why not just initialize at the class member level? @Before is setup before each test and is the same as class level because JUnit creates new class for each test.

Reply:
Done